### ISSUE TITLE
padding has incorrect dimension for `nn.AvgPool2d`'s `def __init__()`

### ISSUE THREAD

**Issue by dyollb** [Project-MONAI/model-zoo#776]
https://github.com/Project-MONAI/model-zoo/blob/d86a1197990805b62502654734b6ec07a02f55b4/models/wholeBrainSeg_Large_UNEST_segmentation/scripts/networks/nest/utils.py#L361

Running mypy yields `arg-type` error.

See https://docs.pytorch.org/docs/stable/generated/torch.nn.AvgPool2d.html#avgpool2d

**Comment by ericspod** [comment]
@tangy5 I think this bundle was contributed by you, could you please have a look at this? Thanks!
